test/functional/tests/io_class/test_io_class_occupancy.py

```diff
@@ -273,7 +274,7 @@ def test_ioclass_occupancy_directory_read(io_size_multiplication, cache_line_siz
             )
 
             # Divergency may be casued be rounding max occupancy
-            if actuall_occupancy > occupancy_limit + Size(100, Unit.Blocks4096):
+            if actuall_occupancy > occupancy_limit * 1.01:
```

Comment:
what's the reasoning here? (I acknowledge that the previous value was no better)

Reply:
When calculating a size of an ioclass in a particular moment CAS doesn't track if IOs that are in flight will be assigned to an ioclass, thus in case of a heavy workload max ioclass size might by slightly exceeded.